docs/questions/query-builder/expressions/now.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Now
+---
+
```

Comment:
@calherries  

Could you give this doc a skim for correctness?

The "Limitations" section in particular.

We put it through one review when we did the datetime docs for 45, so just want to make sure this info is up to date.

Thanks :)

Reply:
Do we have any other functions that don't require an invocation (e.g., `now` works without the parens to call it `now()`.)

Reply:
@jeff-bruemmer we have `count` and `sum` for summarize custom expressions.